linux_kernel_modules/bmi160/bmi160_core.c

```diff
@@ -837,12 +975,24 @@ IIO_CONST_ATTR(in_accel_scale_available,
 static
 IIO_CONST_ATTR(in_anglvel_scale_available,
 	       "0.001065 0.000532 0.000266 0.000133 0.000066");
+static IIO_CONST_ATTR(in_accel_int_available,"0 1");
+static IIO_CONST_ATTR(in_accel_int_anymot_th_min,"0");
+static IIO_CONST_ATTR(in_accel_int_anymot_th_max,"255");
+static IIO_DEVICE_ATTR(in_accel_int, (S_IRUGO | S_IWUSR), show_in_accel_int,
+                       store_in_accel_int, 0);
```

Comment:
I don't love the idea of having the interrupt source set as an integer.  It's not possible for a client to understand what this means without reading the source code of the driver.  I haven't had a chance to look at how other drivers handle this, but my initial feeling is that having text values like "significant_motion"/"any_motion", would be better than 0/1.

Reply:
Ok I'll try to switch to string then !